### ISSUE TITLE
cms files are JSON but his extension is .js

### ISSUE THREAD

**Issue by rkmax** [Gottwik/Enduro#204]
Maybe this is just annoying for IDE people like me, where the IDE try to show the JS syntax but fails because they are json files

![image](https://user-images.githubusercontent.com/395707/34671454-b28609e2-f448-11e7-812e-2ee3f66b9c2e.png)

What do you think to allow both kind of files there, JSON and JS (even more) where the most important thing is the return a valid JSON object

- JSON are allowed write/read (like .settings, .users, etc)
- JS are read only

**Comment by gouldingken** [comment]
I guess they're not valid JSON or JS (JSON would require double quotes around all tokens and values), but I agree they should probably be valid files one way or another! Is there any reason not to just use valid JSON for these files?

**Comment by CosmoMyzrailGorynych** [comment]
@rkmax *.js files in cms folder are not JSON files. They use js, and may sometimes contain dynamic (computed) values. It is especially handy while prototyping. There currently no workarounds to make these objects js-compliant as is, but I might create a pull request today to fix this.

@gouldingken the module you mentioned can be found [here](https://www.npmjs.com/package/require-from-string). imho, writing js objects is more productive that creating pure JSON ones.

**Comment by rkmax** [comment]
So that pull request, it would be look like

```js
//cms/anything.js
module.exports = {
  environment: process.env.NODE_ENV || 'development'
}; // or export default 
```

```json
//cms/anything.json
{
  "environment": "blahblah"
}
```

**Comment by CosmoMyzrailGorynych** [comment]
No.

**Comment by Gottwik** [comment]
merged #214, thank you @CosmoMyzrailGorynych. Published the changed in 1.4.46
@rkmax let us know if this solves your ide issues.

**Comment by rkmax** [comment]
Great!
